GeneralsMD/Code/GameEngine/Source/GameClient/Input/Keyboard.cpp

```diff
@@ -447,8 +447,8 @@ void Keyboard::initKeyNames( void )
 
 			_set_keyname_(L'1',				L'!',				L'\0',	KEY_1  );
 			_set_keyname_(L'2',				L'\"',			L'\0',	KEY_2  );
-			_set_keyname_(L'3',				0x00A3,			L'\0',	KEY_3  );	//�
-			_set_keyname_(L'4',				L'$',				L'�',		KEY_4  );
+			_set_keyname_(L'3',				0x00A3,			L'\0',	KEY_3  );	//£ GBP SIGN
+			_set_keyname_(L'4',				L'$',				L'\x20ac',		KEY_4  ); // € EUR SIGN
```

Comment:
So before it was a Windows Codepage symbol and now it is a Unicode symbol? Does this work correctly?